tests/helpers.py

```diff
@@ -412,7 +426,7 @@ def create_cephfilesystem():
                 selector='app=rook-ceph-mds'
             )
     assert validate_cephfilesystem(fs_name=fs_data['metadata']['name'])
-    return True
+    return CEPHFS_OBJ
```

Comment:
This is a global parameter - why to return it?

Reply:
fixed